jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java

```diff
@@ -109,10 +109,12 @@ public static int getInteger(String attr, WebElement el, int defaultValue) {
   }
 
   private String setNewStyle(String style, int value) {
-    style = style.replaceAll("[-?0-9]+", "");
-    int perInx = style.indexOf("%");
-    return style.substring(0, perInx) + 0 + style.substring(perInx, style.length()-2)
-        + value + style.substring(style.length()-2);
+    String styleLocal;
+    styleLocal= style.replaceAll("[-?0-9]+", "");
+    int perInx;
+    perInx = styleLocal.indexOf("%");
```

Comment:
For what we need to update red line 113 to 2 new green lines 114 and 115?

Reply:
concatenated into one line